packages/clay-label/src/ClayLabel.js

```diff
@@ -0,0 +1,95 @@
+'use strict';
+
+import 'clay-icon';
+import 'clay-link';
+import Component from 'metal-component';
+import Soy from 'metal-soy';
+import { Config } from 'metal-state';
+
+import templates from './ClayLabel.soy.js';
+
+/**
+ * Metal Clay Label component.
+ */
+class ClayLabel extends Component {}
+
+/**
+ * State definition.
+ * @static
+ * @type {!Object}
+ */
+ClayLabel.STATE = {
+	/**
+ 	 * True or false to activate the close button.
+ 	 * @instance
+ 	 * @memberof ClayLabel
+ 	 * @type {?bool}
+ 	 * @default false
+ 	 */
+	closeable: Config.bool().value(false),
+
+	/**
+ 	 * HREF in label.
+ 	 * @instance
+ 	 * @memberof ClayLabel
+ 	 * @type {?string|undefined}
+ 	 * default undefined
+ 	 */
+	href: Config.string(),
+
+	/**
+	 * Id to be applied to the element.
+	 * @instance
+	 * @memberof ClayLabel
+	 * @type {?string|undefined}
+	 * default undefined
+	 */
+	id: Config.string(),
+
+	/**
+	 * The label of the badge element.
+	 * @instance
+	 * @memberof ClayLabel
+	 * @type {?string|undefined}
+	 * default undefined
+	 */
+	label: Config.string(),
+
+	/**
+ 	 * Label size small.
+ 	 * @instance
+ 	 * @memberof ClayLabel
+ 	 * @type {?bool}
+ 	 * @default false
+ 	 */
+	small: Config.bool().value(false),
```

Comment:
Use `size: Config.oneOf(['sm'])` for consistency.

Reply:
You're right.